features/networking/ovn.feature

```diff
@@ -0,0 +1,211 @@
+    Then the expression should be true> @result[:parsed]['end']['sum']['packets'].to_f > 0
+    Then the expression should be true> @result[:parsed]['end']['sum']['jitter_ms'].to_f < 1
+    And I run the :logs client command with:
+      | resource_name | iperf-server |
+    Then the step should succeed
+    And the output is parsed as JSON
+    Then the expression should be true> @result[:parsed]['end']['sum']['lost_percent'].to_f < 10
+    # server doesn't count bytes
+    Then the expression should be true> @result[:parsed]['end']['sum']['packets'].to_f > 0
+    Then the expression should be true> @result[:parsed]['end']['sum']['jitter_ms'].to_f < 1
+
+
+  # @author rbrattai@redhat.com
+  # @case_id OCP-26089
+  @admin
+  @destructive
+  Scenario: New raft leader should be elected if existing leader gets deleted or crashed in hybrid/non-hybrid clusters
+    Given admin uses the "openshift-ovn-kubernetes" project
+    When I store the ovnkube-master "north" leader pod in the clipboard
+    Then the step should succeed
+    Given admin ensures "<%= cb.north_leader.name %>" pod is deleted from the "openshift-ovn-kubernetes" project
+    Then the step should succeed
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I store the ovnkube-master "north" leader pod in the :new_north_leader clipboard
+    Then the step should succeed
+    And the expression should be true> cb.north_leader.name != cb.new_north_leader.name
+    """
+    And admin waits for all pods in the project to become ready up to 60 seconds
+
+
+  # @author rbrattai@redhat.com
+  # @case_id OCP-26091
+  @admin
+  @destructive
+  Scenario: New raft leader should be elected if SB db on existing master is crashed
+    Given admin uses the "openshift-ovn-kubernetes" project
+    When I store the ovnkube-master "south" leader pod in the clipboard
+    Then the step should succeed
+    When the OVN "south" database is killed on the "<%= cb.south_leader.node_name %>" node
+    Then the step should succeed
+
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I store the ovnkube-master "south" leader pod in the :new_south_leader clipboard
+    Then the step should succeed
+    And the expression should be true> cb.south_leader != cb.new_south_leader
+    """
+    And admin waits for all pods in the project to become ready up to 60 seconds
+
+
+  # @author rbrattai@redhat.com
+  # @case_id OCP-26138
+  @admin
+  @destructive
+  Scenario: Inducing Split Brain in the OVN HA cluster
+    Given admin uses the "openshift-ovn-kubernetes" project
+    When I store the ovnkube-master "south" leader pod in the clipboard
+    Then the step should succeed
+
+    Given I select a random master not named "<%= cb.south_leader.node_name %>"
+    # don't block all traffic that breaks etcd, just block the OVN ssl ports
+    When I run commands on the host:
+      | iptables -t filter -A INPUT -s <%= cb.south_leader.ip %> -p tcp --dport 9641:9648 -j DROP |
```

Comment:
@rbbratta I think we should be specific to use only 9643 and 9644 to focus on just raft isolation? Also not sure what are 9645-9648 used for. May be in good case we just need range from 9641:9648 or in best case it would be 9643:9644. Anyways let see wait to see what your experiments tell.

Reply:
I originally thought it was just 9641 and 9642 from dumping oc describe pods, but that didn't work so I tried a bunch of ports up to 9648, but then I found the exact ports in CNO.

I guess that makes sense to leave 9641 and 9642 open.  I'll re-test.

Reply:
updated for 9643-9644, done